druid/src/event.rs

```diff
@@ -119,16 +123,28 @@ pub enum Event {
     /// intensive in response to an `AnimFrame` event: it might make Druid miss
     /// the monitor's refresh, causing lag or jerky animation.
     AnimFrame(u64),
+    /// An event containing a [`Command`] to be handled by the widget.
+    ///
+    /// [`Command`]s are messages, optionally with attached data, that can
+    /// may be generated from a number of sources:
+    ///
+    /// - If your application uses  menus (either window or context menus)
+    /// then the [`MenuItem`]s in the menu will each correspond to a `Command`.
+    /// When the menu item is selected, that [`Command`] will be delivered to
+    /// the root widget of the appropriate window.
+    /// - If you are doing work in another thread (using an [`ExtEventSink`])
+    /// then [`Command`]s are the mechanism by which you communicate back to
+    /// the main thread.
+    /// - Widgets and other Druid components can send custom [`Command`]s at
+    /// runtime, via methods such as [`EventCtx::submit_command`].
     /// Called with an arbitrary [`Command`], submitted from elsewhere in
     /// the application.
```

Comment:
Did you mean to leave this sentence in? It looks redundant, given the new part.

Reply:
Good catch!